ui/src/app/utils/access-utils.tsx

```diff
@@ -279,9 +293,23 @@ export const computeDisplayDates = ({completionEpochMillis, expirationEpochMilli
   );
 };
 
-// return true if user is egligible for registered tier.
-// A user loses tier eligiblity when they are removed from institution tier requirement
+// return true if user is eligible for registered tier.
+// A user loses tier eligibility when they are removed from institution tier requirement
 export const eligibleForRegisteredForTier = (tierEligiblities: Array<UserTierEligibility>): boolean => {
   const rtEligiblity = tierEligiblities.find(t => t.accessTierShortName === AccessTierShortNames.Registered)
   return !!rtEligiblity && rtEligiblity.eligible
 };
+
+// Show eraCommons module only if institution has registered tiered and eraCommon is not required
```

Comment:
we're going to need this for CT as well, so I'd prefer a version of these which works for any tier

also: "not" is not correct here